### PR TITLE
Simplifies (and fixes) tags

### PR DESCRIPTION
**Pull request by Super3222**
Fixes https://github.com/yogstation13/yogstation/issues/599
Fixes https://github.com/yogstation13/yogstation/issues/426

:cl: Super3222
rscadd: Tags are smaller. Yay
/:cl:

### CODE REVIEW

**Comment by FluffySurvivor** [comment]
best pr 2016

File: code/modules/client/verbs/ooc.dm [hunk 1]
```diff
@@ -77,7 +77,7 @@
 
 			else if(!(key in C.prefs.ignoring))
 				if(is_donator(src))
-					C << "<font color='[normal_ooc_colour]'><span class='ooc'><span class='prefix'>\[Donator\] OOC:</span> <EM>[keyname]:</EM> <span class='message'>[msg]</span></span></font>"
+					C << "<font color='[normal_ooc_colour]'><span class='ooc'><span class='prefix'>\[D\] OOC:</span> <EM>[keyname]:</EM> <span class='message'>[msg]</span></span></font>"
```
Comment (oisin100) [inline code/modules/client/verbs/ooc.dm]: This is a bit too simple...

**Comment by oisin100** [comment]
This is too simple, I would rather have the old system then have tags this simple

**Comment by Super3222** [comment]
Fair enough. I was thinking of simplifying them to make it easier to read like /tg/'s version without [overlord of the server] before someones ckey.

**Comment by Alek2ander** [comment]
Why do donators have shitty tags instead of icons?

**Comment by Super3222** [comment]
Icons like what.
The byond icon was/is for byond members.

**Comment by Cruix** [comment]
Yay/Nay in 24 hours or this is being closed.

Or preferably just remove the abbreviations so I can merge just the fixes.

**Pull request closed without merging**